### PR TITLE
add associations

### PR DESCRIPTION
**Pull request by TracyMuso**
- create the remaining classes for our school library and create the associations between them
- create Classroom, Rental, and Book classes
- Create the has-many side of Book and Rental (a book has many rentals).
- Create the belongs-to side of Rental and Book (a rental belongs to a book).
- Create the has-many side of Person and Rental (a person has many rentals).
- Create the belongs-to side of Rental and Person (a rental belongs to a person).

### CODE REVIEW

File: rental.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+    person.rental << self
+    book.rental << self
```
Comment (DammyShittu) [inline rental.rb]: - [ ] I think these definitions are wrong. They should be `rentals`, not `rental` based on what you defined in your respective Book and Person classes.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @TracyMuso,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: rental.rb (hunk 1 above)
Comment (TracyMuso) [reply rental.rb]: Thanks, I have corrected it

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @TracyMuso , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**